src/ansys/simai/core/data/models.py

```diff
@@ -108,3 +109,22 @@ def build(
                 dismiss_data_with_volume_overflow,
             )
         )
+
+    def cancel_build(self, model: Identifiable[Model]):
+        """Cancel an ongoing build given the model.
+
+        Args:
+            model: a model object or model id whose ongoing build to cancel
+
+        .. code-block:: python
+
+            simai.models.cancel_build(new_model)
+        """
+
+        model_object = get_object_from_identifiable(model, self._client.models)
+        response = self._client._api.cancel_build(model_object.project_id)
+        return (
```

Comment:
what would the user do with the return strings? print them? Wouldn't it be more intuitive if just a boolean was returned?